resources/carrier/en/44.txt

```diff
@@ -13,6 +13,9 @@
 # limitations under the License.
 
 # Carrier details updated from http://www.ofcom.org.uk/static/numbering/S7.xls
+# Carrier names have been updated to friendlier trading/brand names
+# The mapping of company names can be found in the project below.
```

Comment:
I would prefer an explicit mapping here, because your project may have a shorter lifetime than this one, who knows :) it's not so big?
Also if someone else contributes at any point, they may want to add a mapping.